core/modules/views_ui/views_ui.admin.inc

```diff
@@ -2144,9 +2144,7 @@ function views_ui_edit_view_form_cancel($form, &$form_state) {
   tempstore_clear('views.view', $form_state['view']->name);
   backdrop_set_message(t('Any changes to the %name view have been discarded.', array('%name' => $form_state['view']->get_human_name())), 'info');
   if (empty($form['view']->vid)) {
-    // I seem to have to backdrop_goto here because I can't get fapi to
-    // honor the redirect target. Not sure what I screwed up here.
-    backdrop_goto('admin/structure/views');
```

Comment:
In my testing using `$form_state['redirect']` works perfectly fine here. This code comment is ancient, this may have been fixed as part of converting views to core, which reworked the save handling.